CHANGELOG.md

```diff
@@ -37,6 +37,9 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 ### Added
 - New detector `ResourceInMultipleThreadsDetector` and introduced new bug type:
   - `AT_UNSAFE_RESOURCE_ACCESS_IN_THREAD` is reported in case of unsafe resource access in multiple threads.
+- New detector `AtomicOperationsCombinedDetector` and introduced new bug types:
+  - `AT_COMBINED_ATOMIC_OPERATIONS_ARE_NOT_ATOMIC` is reported in case of combined atomic operations are not synchronized.
+  - `AT_ATOMIC_OPERATION_NEEDS_SYNCHRONIZATION` is reported when an atomic operation is not synchronized, but should be because of thread safety.
```

Comment:
It looks like this Added block got here by mistake when merging, as there is already an Added section in this Unreleased part. Can you please move this entry there after updating master?